room.py

```diff
@@ -0,0 +1,90 @@
+from enum import Enum
+from deck import Deck
+
+
+class room:
+    class players(Enum):
+        p1 = 1
+        p2 = 2
+        p3 = 3
+        p4 = 4
+
+    class Teams(Enum):
+        Blue = 1
+        Red = 2
+
+    class bid_status(Enum):
+        Pass = 1
+        Double = 2
+        Bid = 3
+
+    def __init__(self, user1, user2, user3, user4):
+        self.player = {
+            self.players.p1: user1,
+            self.players.p2: user2,
+            self.players.p3: user3,
+            self.players.p4: user4
+        }
+        self.highest_bidder = None
+        self.current_bidder = None
+        self.trump_card = None
+        self.current_trump = None
+        self.contract = None
+        self.current_contract = None
+        self.passes = 0  # when passes reaches to 3 bis is finalized
+        self.doubles = 0
+        self.bid_functions = {
+            room.bid_status.Pass: self.set_pass,
+            room.bid_status.Double: self.set_double,
+            room.bid_status.Bid: self.set_bid
+        }
+        self.deck = Deck()
+        self.deck.shuffle()
+
+        cards_distributed = self.deck.distribute()
+        self.teammates = {
+            self.players.p1: self.players.p3,
+            self.players.p2: self.players.p4,
+            self.players.p3: self.players.p1,
+            self.players.p4: self.players.p2
+        }
+        self.teams = {
+            self.players.p1: self.Teams.Blue,
```

Comment:
Should we move the team-name for a player to the definition of the Player itself ?